Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ghcr.io/celestiaorg/celestia-app:v1.3.0 AS celestia-app
 
-FROM ghcr.io/celestiaorg/celestia-node:v0.12.0
+FROM opcelestia/celestia-da:v0.12.0
```

Comment:
where is this hosted?

Reply:
This will be replaced with `ghcr.io/celestiaorg/celestia-da:v0.12.0`

Reply:
New release of `celestia-da` should trigger this after: https://github.com/rollkit/celestia-da/pull/20 is merged